db_stress_tool/db_stress_test_base.cc

```diff
@@ -1642,6 +1657,8 @@ Status StressTest::TestCheckpoint(ThreadState* thread,
       FLAGS_db + "/.checkpoint" + ToString(thread->tid);
   Options tmp_opts(options_);
   tmp_opts.listeners.clear();
+  db_stress_listener_ = new DBStressCompactionListener();
+  tmp_opts.listeners.emplace_back(db_stress_listener_);
```

Comment:
I think we don't need to do the tracking for these auxiliary functions (backup/restore/checkpoint).